### DISCUSSION TITLE
change sidebar langugae setting

### DISCUSSION THREAD

**Discussion by 007Poe** [CollaboraOnline/online#3679] [Show and tell]
Please Help me!!!
How can i change the sidebar language setting from English to Japanese?

**Comment by pedropintosilva** [comment]
Hi @007Poe !

Could you please start this discussion in the forum instead? https://forum.collaboraonline.com/ also please provide there:
* What version of Collabora Online are you running
* What integrator are you using (Nextcloud, ownCloud etC)

Thanks!

**Comment by thebearon** [comment]
For anyone stumbling across this, see:
https://sdk.collaboraonline.com/docs/installation/Installation_from_packages.html#localization
